frontend/src/codegen/model/current-values.ts

```diff
@@ -0,0 +1,20 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * Flathub API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 0.1.0
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+/**
+ *
+ * @export
+ * @interface CurrentValues
+ */
+export interface CurrentValues {}
```

Comment:
@kymckay any idea why this would generate wrong like this? I would probably just blame python :/

Reply:
@razzeee 

Few possible causes:
- The backend type can't be expressed in openapi spec (seems unlikely)
- FastAPI fails to express the backend type in the openapi spec
- OpenAPI Generator fails to support the way this type is expressed in the openapi spec

I'd check how the type appears in `openapi.json`, if that looks legit then it's probably the latter of those options - which seems to be the case here:

```json
"RequestData": {
  "properties": {
    "keys": {
      "additionalProperties": {
        "anyOf": [
          {
            "type": "string"
          },
          {
            "type": "null"
          }
        ]
      },
      "type": "object",
      "title": "Keys"
    },
    "current_values": {
      "additionalProperties": {
        "anyOf": [
          {
            "type": "string"
          },
          {
            "type": "null"
          }
        ]
      },
      "type": "object",
      "title": "Current Values"
    }
  },
  "type": "object",
  "required": [
    "keys",
    "current_values"
  ],
  "title": "RequestData"
},
```

Reply:
Docs on https://swagger.io/docs/specification/data-models/data-types/ (search for `null`) suggest, that it's generated wrong? Then again, it might not be valid in that case

Reply:
Then again, seems like that would also be buggy in the generator https://github.com/OpenAPITools/openapi-generator/issues/13445

Reply:
Good find, so FastAPI seems the likely culprit. Could test grabbing the json and editing this to use `nullable` for the string type to see if codegen then works.

Reply:
Doesn't seem to affect, what's generated at all, as far as I could test it